sherpa/astro/ui/utils.py

```diff
@@ -654,7 +677,12 @@ def _get_show_bkg_source(self, id=None, bkg_id=None):
 
         return model_str
 
-    def show_bkg(self, id=None, bkg_id=None, outfile=None, clobber=False):
+    def show_bkg(self,
+                 id: Optional[IdType] = None,
+                 bkg_id: Optional[IdType] = None,
+                 outfile=None,
```

Comment:
```suggestion
                 outfile: str = None,
```

Reply:
Same below. I'm sure @DougBurke had a good reason to not type this, so I want to discuss here instead of making suggestions for all of them at this point.

Reply:
Now, I did work on this in a recent PR 

- #2054 
 
 and came up with

https://github.com/sherpa/sherpa/blob/307ddbbd4b7ff98e3c9720915f05aa530f4a1262/sherpa/fit.py#L570-L579

so maybe I can use that.

Reply:
But not in this PR.

Reply:
OK.